### PR TITLE
[FEATURE] Display total active nodes

### PR DESCRIPTION
**Pull request by adibas03**

### CODE REVIEW

File: app/stores/KeenStore.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const COLLECTIONS = {
 };
 
 // 5 minutes in milliseconds
-const ACTIVE_TIMENODES_POLLING_INTERVAL = 5 * 60 * 1000;
+const ACTIVE_TIMENODES_POLLING_INTERVAL = 2 * 60 * 1000;
```
Comment (Bagaric) [inline app/stores/KeenStore.js]: Please update the comment above to match the code.

File: app/stores/TimeNodeStore.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ class SIGNATURE_ERRORS {
 }
 
 // 1 minute as milliseconds
-const STATUS_UPDATE_INTERVAL = 4 * 60 * 1000;
+const STATUS_UPDATE_INTERVAL = 2 * 60 * 1000;
```
Comment (Bagaric) [inline app/stores/TimeNodeStore.js]: Please update the comment as well.

File: app/stores/TimeNodeStore.js [hunk 3]
```diff
@@ -82,6 +82,7 @@ export default class TimeNodeStore {
       logLevel: 1,
       milliseconds: 15000,
       autostart: false,
+      analysis: 'on',
```
Comment (Bagaric) [inline app/stores/TimeNodeStore.js]: What does this parameter do? Is this parameter option already merged into the eac.js-client?

Comment (adibas03) [reply app/stores/TimeNodeStore.js]: it is now irrelevant, as the analytics was not implemented at the eac-client level

**Comment by adibas03** [comment]
@Bagaric Implemented updates

**Pull request merged**